README.md

```diff
@@ -11,3 +12,41 @@ The BM3D ORNL code is designed to be easy to use and easy to integrate into exis
 The BM3D ORNL code is released under an open-source license, and is freely available for download and use.
 
 For more information, check out our [FAQ](docs/FAQ.md).
+
+How to install
+--------------
+
+For **users**, you can install the latest version published on our anaconda channel with:
+
+```bash
+conda install neutronimaging::bm3dornl
+```
+
+Or use `pip` to install from PyPI
+
+```bash
+pip install bm3dornl
+```
+
+Since the `PyPI` version relies on pre-built `CuPy` from PyPI, it is possible that the bundled latest version of `CuPy` might not be compatible.
+In such situations, please either install the correct pre-built version via pip, e.g. `pip install cupy-cuda11x` for Nvidia card with older drivers, or use local `nvcc` to build `CuPy` from source with `pip install cupy`.
+
+How to contribute
+-----------------
+
+For **developers**, please fork this repo and create a conda development environment with
+
+```bash
+conda env create -f environment.yml
+```
+
+followed by
+
+```bash
+pip install --no-deps -e .
+```
+
+The option `--no-deps` here is critical as `pip` will try to install a second set of dependencies based on information from `pyproject.toml`.
```

Comment:
This is an excellent observation/comment. Can you copy it into the python template?

Reply:
will do